src/sagas/projects.js

```diff
@@ -47,6 +50,17 @@ export function* changeCurrentProject() {
   yield call(saveCurrentProject, state);
 }
 
+export function* rehydrateProject({payload: {rehydratedProject}}) {
```

Comment:
Are you down to add some tests for this as well as a test for `applicationLoaded` that asserts that it’s called if there is a rehydrated project?